packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js

```diff
@@ -75,15 +75,15 @@ const links = [
       { id: 'NATIVE_BANNER_ALERT', route: routes.NATIVE_BANNER_ALERT, children: 'Banner alerts' },
       { id: 'NATIVE_BUTTON', route: routes.NATIVE_BUTTON, children: 'Button' },
       { id: 'NATIVE_CARDS', route: routes.NATIVE_CARDS, children: 'Cards' },
+      { id: 'NATIVE_HORIZONTAL_NAV', route: routes.NATIVE_HORIZONTAL_NAV, children: 'Horizontal navigation' },
       { id: 'NATIVE_ICONS', route: routes.NATIVE_ICONS, children: 'Icons' },
       { id: 'NATIVE_INPUT', route: routes.NATIVE_INPUT, children: 'Text input' },
       { id: 'NATIVE_SPINNER', route: routes.NATIVE_SPINNER, children: 'Spinners' },
-      { id: 'NATIVE_HORIZONTAL_NAV', route: routes.NATIVE_HORIZONTAL_NAV, children: 'Horizontal navigation' },
+      { id: 'NATIVE_SWITCH', route: routes.NATIVE_SWITCH, children: 'Switch' },
```

Comment:
Again, this should be plural. Btw there is sort function on the docs nav so you don't need to manually sort these ... 😛

Reply:
Haha, I know it sorts them manually but I couldn't leave it unsorted in the code! I realise that's ridiculous though haha